packages/server/worker/src/lib/executors/flow-job-executor.ts

```diff
@@ -140,6 +140,10 @@ async function executeFlow(
       await handleTimeoutError(jobData, engineToken);
```

Comment:
as we discussed offline, timeouts are currently marked as successful in the queue. We're not changing it for now.